_posts/2022-07-20-developer-cli.markdown

```diff
@@ -28,6 +28,8 @@ You may enable auto-completion for the CLI by running the following command:
 ```shell
 flotta completion bash > /usr/share/bash-completion/completions/flotta
 ```
+( If the above command didn't work use `| sudo tee` instead of  `> ` )
```

Comment:
What do you think to state that the command can fail because of permission issues (instead of a generic "didn't work")? 
In my opinion, It can be useful for a new user to understand what problem your change is resolving.

Reply:
Also, what do you think about mentioning the full command instead of explaining the replacement?
```suggestion
Or (if the above command failed with permission denied error):
flotta completion bash | sudo tee /usr/share/bash-completion/completions/flotta
```

Reply:
Thank you, for the suggestions!